### PR TITLE
Logs merge. Enterprise to master.

### PR DESCRIPTION
**Pull request by AdamZielinski-Devtactics**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -57,16 +57,16 @@ GIT
 GEM
   remote: https://rubygems.org/
   specs:
-    HDRHistogram (0.1.9)
-    activesupport (7.0.7.2)
+    HDRHistogram (0.1.3)
+    activesupport (5.2.8)
```
Comment (gbarc-dt) [inline Gemfile.lock:61]: why is it downgraded from 7?

Comment (gbarc-dt) [reply Gemfile.lock:61]: actually - why all gems are downgrade, pls use the ones from master

Comment (AdamZielinski-Devtactics) [reply Gemfile.lock]: Gemfile.lock has now been recreated according to current Gemfile

File: Dockerfile.tcie [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+RUN ( \
+  apt-get update; \
+  apt-get upgrade -y ; \
+  apt-get install -y build-essential git curl  libpq-dev; \
+  bundle install --deployment; \
+  apt-get remove -y build-essential git curl gcc g++ make perl; \
+  apt-get -y autoremove; \
+  rm -rf /var/lib/apt/lists/* ; \
+  bundle clean && rm -rf /app/vendor/bundle/ruby/2.5.0/cache/*; \
```
[the remark this answers is not in the document]
Comment (AdamZielinski-Devtactics) [reply Dockerfile.tcie]: Fixed.

File: Gemfile [hunk 3]
```diff
@@ -13,6 +13,8 @@ gem 'coder', github: 'piotrmilcarz/coder'
 gem 'concurrent-ruby', require: 'concurrent'
 gem 'connection_pool'
 gem 'dalli'
+gem 'google-protobuf', '~> 3.19.6'
+gem 'jemalloc', git: gh('joshk/jemalloc-rb')
```
[the remark this answers is not in the document]
Comment (AdamZielinski-Devtactics) [reply Gemfile]: Fixed.

File: Gemfile.lock [hunk 4]
```diff
@@ -1750,8 +1866,9 @@ DEPENDENCIES
   panolint
   pg
   pry
-  puma
+  puma (~> 4)
```
Comment (GbArc) [inline Gemfile.lock:1869]: puma was 6, pls don't downgrade

File: Gemfile.lock [hunk 5]
```diff
@@ -1704,30 +1820,29 @@ GEM
       simplecov_json_formatter (~> 0.1)
     simplecov-html (0.12.3)
     simplecov_json_formatter (0.1.4)
-    sinatra (3.1.0)
+    sinatra (4.0.0)
```
Comment (GbArc) [inline Gemfile.lock:1823]: this may hurt, better keep the 3.x for sinatra

File: Gemfile.lock (hunk 4 above)
Comment (AdamZielinski-Devtactics) [reply Gemfile.lock]: fixed

File: Gemfile [hunk 6]
```diff
@@ -13,6 +13,7 @@ gem 'coder', github: 'piotrmilcarz/coder'
 gem 'concurrent-ruby', require: 'concurrent'
 gem 'connection_pool'
 gem 'dalli'
+gem 'google-protobuf', '~> 3.19.6'
```
[the remark this answers is not in the document]
Comment (AdamZielinski-Devtactics) [reply Gemfile]: Old constraint removed, now its 3.25, newest 3.x version. Should it be locked down to 3.24?

File: Gemfile.lock (hunk 5 above)
Comment (AdamZielinski-Devtactics) [reply Gemfile.lock]: Locked down to 3.x.

**Comment by DominikAlberski** [comment]
copy of #271 merged in #272

**Pull request closed without merging**